pyproject.toml

```diff
@@ -1,22 +1,24 @@
 # The `dev-dependencies` section sets up tools for building `jj`
-# documentation. `poetry` will install these in a virtual environment.
-[tool.poetry]
-name = "jj-docs"
-package-mode = false
+# documentation. `uv` will install these in a virtual environment.
 
-[tool.poetry.dependencies]
-python = "^3.8"
+[project]
```

Comment:
This are all minor changes, but I'd apply this diff (from my version of the uv PR):

```diff
 [project]
 name = "jj-docs"
+description = "Sets up MkDocs and related Python tools"
+requires-python = ">=3.9"
 version = "0.1.0"
-requires-python = ">=3.8"
 dependencies = []

 [tool.uv]
+package = false
```

Python 3.8 is no longer supported, though if you prefer to keep it, I don't know if it causes any harm.